VK/VK/Services/Models/Size.swift

```diff
@@ -13,4 +13,6 @@ struct Size: Decodable {
     let width: Int
     /// Ссылка на изображение
     let url: String
+
+    var aspectRatio: CGFloat { CGFloat(height) / CGFloat(width) }
```

Comment:
```suggestion
    var aspectRatio: CGFloat { CGFloat(height) / CGFloat(width) }
```
нет защиты от деления на 0